FCAppfront/src/component/SelectUserAuto.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import { getUsersFn } from '../api/endpoints';
+import { SelectChangeEvent } from '@mui/material';
+
+export default function SelectUserAuto(props: {
+    selectedClient ?: {label: string, id: number};
+    setSelectedClient : (label:string, id:number) => void
+}) {
+    const [People, setPeople] = React.useState<{id_klient: number, imie:string, nazwisko: string}[]>([])
```

Comment:
make 
```
{
  id_klient: number,
  imie:string,
  nazwisko: string
}
 ```
a type, then use it here, it will help you handle this type throughout app